docs/docs/reference/release-notes/v4.15/index.md

```diff
@@ -25,6 +25,8 @@ Also check out [notable bugfixes](#notable-bugfixes--improvements).
 
 We're releasing a built-in `<Script>` component that aids in loading third-party scripts performantly.
 
+> Note - If you are using Jest, you will need to include `gatsby-script` in your `transformIgnorePatterns` key in your Jest config since `gatsby-script` is an ES module. See [the unit testing documentation on Jest configuration](/docs/how-to/testing/unit-testing/#2-creating-a-configuration-file-for-jest) for more details.
```

Comment:
Let's move this to the end of the announcement, it doesn't need to be so prominent at the beginning (not everyone will run into this)